src/Error/Error.php

```diff
@@ -142,11 +142,7 @@ private function updateRepr(): void
         }
 
         if ($this->name) {
-            if (\is_string($this->name) || $this->name instanceof \Stringable) {
-                $name = \sprintf('"%s"', $this->name);
-            } else {
-                $name = json_encode($this->name);
-            }
```

Comment:
This is weird.

The property is typed as `private ?string $name;` so it always a string in this `if`.

Reply:
I've just had a quick look to see why this is done this way and this is indeed not valid anymore.